grakn-orientdb-factory/src/test/java/ai/grakn/factory/GraknOrientDBGraphFactoryTest.java

```diff
@@ -52,13 +53,15 @@ public void clear() throws GraknValidationException {
         graph.clear();
     }
 
+    @Ignore
```

Comment:
For future reference if you put the `@Ignore` at the top of this class then all tests are ignored.